lib/build/index-static.jsx

```diff
@@ -38,7 +38,7 @@ function generateWithPageHTML(url, options, pageHTML) {
   options = _.defaults(options || {}, {
     meta: {}
   });
-  var locale = url.split('/')[0];
+  var locale = options.locale || 'en-US';
```

Comment:
not for regular cases, but testing might and I don't know for sure since I haven't removed them...